docs/docs/addons-perf.md

```diff
@@ -14,6 +14,10 @@ next: test-utils.html
 import Perf from 'react-addons-perf' // ES6
 var Perf = require('react-addons-perf') // ES5 with npm
 var Perf = React.addons.Perf; // ES5 with react-with-addons.js
+
+//If Perf is not accessible from console i.e throws RefrenceError: 
+//Perf is not defined, create a global 
+window.Perf = Perf;
```

Comment:
In this tutorial I am using only one file but in real projects developers use bable and webpack or other such tools. If we import Perf ES6 style in any file of our project we have to make it global to access it from console and @gaearon has mentioned it in issue #6174 . So I think we should let developers know solution to a common problem beforehand. What do you think ?